src/classes/GE_LookupController.cls

```diff
@@ -60,10 +101,34 @@ public with sharing class GE_LookupController {
                 results.add(new LookupResult(queryResult));
             }
         }
+        return results;
+    }
+
+    /**
+     * Search for an SObject by its name field using SOQL
+     * Objects without Name fields (Case, Task, Event...) are not compatible with this component.
+     *
+     * @param searchValue   Value to search for
+     * @param sObjectType   Type of SObject to Search
+     *
+     * @return              A List<LookupResult> containing records that match the search criteria.
+     */
+    @TestVisible
+    private static List<LookupResult> doSearchSOQL(String searchValue, String sObjectType) {
+        String formattedValue = '%' + searchValue + '%';
```

Comment:
Should we escape the searchValue here like you have in the other queries?

Reply:
This one's ok since we're using it as a bound variable in the query rather than building a string with it.